plugin/completion/lib_complete.py

```diff
@@ -198,6 +203,115 @@ def complete(self, completion_request):
         log.debug(' completions: %s' % completions)
         return (completion_request, completions)
 
+    @staticmethod
+    def location_from_type(clangType):
+        """ Return location from type.
```

Comment:
Wrong documentation style. No spaces should be before `Return` here. Also inputs and returns not documented.